stubs/tensorflow/tensorflow/keras/layers/__init__.pyi

```diff
@@ -442,4 +442,21 @@ class GaussianDropout(Layer[tf.Tensor, tf.Tensor]):
         name: str | None = None,
     ) -> None: ...
 
+class MaxPooling2D(Layer[tf.Tensor, tf.Tensor]):
```

Comment:
In the implementation this doesn't extend Layer directly, it extends BasePooling

https://github.com/keras-team/keras/blob/v3.3.3/keras/src/layers/pooling/base_pooling.py